train_model/requirements.txt

```diff
@@ -1,2 +1,3 @@
 mongoengine
 python-dotenv
+nirjas
```

Comment:
Will be nice to add Nirjas to [setup.py's install_requires](https://github.com/fossology/FOSSologyML/blob/master/setup.py#L45) as it will be required by rigel cli and server.